www/assets/css/keebs.css

```diff
@@ -56,6 +80,118 @@
   margin: 0;
   text-wrap: balance;
   text-align: center;
+  display: flex;
+  position: relative;
+}
+
+.keyboard form fieldset input {
+  border-radius: 0px;
+  border-top: 1px solid #6668;
+  border-bottom: 1px solid #6668;
+  box-sizing: border-box;
+  border: none;
+  z-index: 1;
+  color: var(--fg-normal);
+}
+
+.keyboard form fieldset::before{
```

Comment:
This section is for styling the geometry/characters selectors (I wanted to try something very light, w/o colors, with a nice CSS animation to highlight the selected one).